pydis_site/apps/resources/views/resources.py

```diff
@@ -23,12 +23,15 @@ def resource_view(request: HttpRequest) -> HttpResponse:
         )
     }
 
+    topics = sorted(RESOURCE_META_TAGS.get("topics"))
+
     return render(
         request,
         template_name="resources/resources.html",
         context={
             "checkboxOptions": checkbox_options,
-            "topics": sorted(RESOURCE_META_TAGS.get("topics")),
+            "topics_1": topics[:len(topics) // 2],
+            "topics_2": topics[len(topics) // 2:],
```

Comment:
We need to double check that this doesn't affect the back end logic in any way.

Reply:
It doesn't